app/concerns/listens.rb

```diff
@@ -95,24 +97,24 @@ def play_local_mp3
     headers["Cache-Control"] = "public, must-revalidate, max-age=0"
     headers["Pragma"] = "no-cache"
     headers['Connection'] = 'close'
-    if !request.headers["Range"] or request.headers["Range"]=="bytes=0-"# browser wants the whole file
+    if !request.headers["Range"] || (request.headers["Range"] == "bytes=0-") # browser wants the whole file
       status = "200 OK"
       headers["Content-Length"] = (file_end.to_i - file_begin.to_i + 1).to_s
 
-      send_file file_to_send, :type => 'audio/mpeg', :disposition => 'attachment;',
-      :url_based_filename => true, :status => status, :stream => true, :buffer_size  =>  4096
+      send_file file_to_send, type: 'audio/mpeg', disposition: 'attachment;',
+                              url_based_filename: true, status: status, stream: true, buffer_size: 4096
     else
```

Comment:
Personally not a fan of this style of indentation, is this something we can disable for now or set to two spaces? (vs. aligned with the first argument)